sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -150,7 +153,7 @@ def to_header(self): # pylint: disable=no-self-use
         :return: A key value pair dictionary
         """
         temp_headers = {} # type: Dict[str, str]
-        inject(_set_headers_from_http_request_headers, temp_headers)
+        inject(temp_headers)
```

Comment:
I don't quite understand what the purpose of this is?

Reply:
it's to propagate values to a http client